_sass/_base.scss

```diff
@@ -0,0 +1,206 @@
+/**
```

Comment:
It's a Jekyll standard :P. But I have no preference among CSS pre processors and their extensions. However I use more Stylus. :)